server/controllers/mailList/index.js

```diff
@@ -0,0 +1,30 @@
+const {
+  mailList: { addMail, getEmails },
+} = require('../../models/queries/');
+const { findEmail } = require('../../helpers/findEmail');
+const { validateEmail } = require('../../validation');
+
+exports.addEmail = async (req, res, next) => {
```

Comment:
if you are only exporting one thing then use `module.exports`

Reply:
Ok

Reply:
OK